multi-int-store/src/test/java/resources/test.yml

```diff
@@ -0,0 +1,7 @@
+callback-accept-version: 0.1.0-SNAPSHOT
+aws:
+  s3:
+    endpointUrl: https://change.me
+    region: us-west-1
+    bucket:
+      quarantine: test-bucket
```

Comment:
✏️ Is this file needed? Isn't everything already mocked in `TestUtilConfig.java`?

Reply:
Good catch, it was left over from my attempts to salvage the the itests. I'll remove it.